### DISCUSSION TITLE
Missing Create Project Button [Jun 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5956] [Discord Threads]
> 🙋‍♂️ **Sapper**  
## 0.107.0-beta.1  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1107906626066858004/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### are you using admin ac?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1107906626335297627)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
